assets/js/googlesitekit/datastore/location/navigation.test.js

```diff
@@ -52,8 +52,7 @@ describe( 'core/location', () => {
 	describe( 'actions', () => {
 		describe( 'navigateTo', () => {
 			it( 'should require a valid URL', () => {
-				return expect( registry.dispatch( STORE_NAME ).navigateTo( 'testurl' ) )
-					.rejects
+				return expect( () => registry.dispatch( STORE_NAME ).navigateTo( 'testurl' ) )
```

Comment:
Note how I've had to update this test though

If I don't wrap the call inside an anonymous function the Jest fails the test because the exception isn't caught by this syntax  (which makes sense as previous the exceptions were getting swallowed thus this ticket)

Instead, I can wrap it in a try-catch block but then that's ugly and prone to false positives!

That does mean that all my other changes are untested though! I had a look through the other tests and they're not testing that `invariant` is catching bad input (which does make sense as previously the code was very basic)

I'm happy to add more tests though